src/hb-aat.h

```diff
@@ -0,0 +1,540 @@
+  HB_AAT_LAYOUT_SELECTOR_EXPERT_CHARACTERS            = 10,
+  HB_AAT_LAYOUT_SELECTOR_JIS2004_CHARACTERS           = 11,
+  HB_AAT_LAYOUT_SELECTOR_HOJO_CHARACTERS              = 12,
+  HB_AAT_LAYOUT_SELECTOR_NLCCHARACTERS                = 13,
+  HB_AAT_LAYOUT_SELECTOR_TRADITIONAL_NAMES_CHARACTERS = 14,
+/*
+ * Selectors for #HB_AAT_LAYOUT_FEATURE_TYPE_NUMBER_CASE
+ *
+ * Since: REPLACEME
+ */
+  HB_AAT_LAYOUT_SELECTOR_LOWER_CASE_NUMBERS = 0,
+  HB_AAT_LAYOUT_SELECTOR_UPPER_CASE_NUMBERS = 1,
+/*
+ * Selectors for #HB_AAT_LAYOUT_FEATURE_TYPE_TEXT_SPACING
+ *
+ * Since: REPLACEME
+ */
+  HB_AAT_LAYOUT_SELECTOR_PROPORTIONAL_TEXT     = 0,
+  HB_AAT_LAYOUT_SELECTOR_MONOSPACED_TEXT       = 1,
+  HB_AAT_LAYOUT_SELECTOR_HALF_WIDTH_TEXT       = 2,
+  HB_AAT_LAYOUT_SELECTOR_THIRD_WIDTH_TEXT      = 3,
+  HB_AAT_LAYOUT_SELECTOR_QUARTER_WIDTH_TEXT    = 4,
+  HB_AAT_LAYOUT_SELECTOR_ALT_PROPORTIONAL_TEXT = 5,
+  HB_AAT_LAYOUT_SELECTOR_ALT_HALF_WIDTH_TEXT   = 6,
+/*
+ * Selectors for #HB_AAT_LAYOUT_FEATURE_TYPE_TRANSLITERATION
+ *
+ * Since: REPLACEME
+ */
+  HB_AAT_LAYOUT_SELECTOR_NO_TRANSLITERATION        = 0,
+  HB_AAT_LAYOUT_SELECTOR_HANJA_TO_HANGUL           = 1,
+  HB_AAT_LAYOUT_SELECTOR_HIRAGANA_TO_KATAKANA      = 2,
+  HB_AAT_LAYOUT_SELECTOR_KATAKANA_TO_HIRAGANA      = 3,
+  HB_AAT_LAYOUT_SELECTOR_KANA_TO_ROMANIZATION      = 4,
+  HB_AAT_LAYOUT_SELECTOR_ROMANIZATION_TO_HIRAGANA  = 5,
+  HB_AAT_LAYOUT_SELECTOR_ROMANIZATION_TO_KATAKANA  = 6,
+  HB_AAT_LAYOUT_SELECTOR_HANJA_TO_HANGUL_ALT_ONE   = 7,
+  HB_AAT_LAYOUT_SELECTOR_HANJA_TO_HANGUL_ALT_TWO   = 8,
+  HB_AAT_LAYOUT_SELECTOR_HANJA_TO_HANGUL_ALT_THREE = 9,
+/*
+ * Selectors for #HB_AAT_LAYOUT_FEATURE_TYPE_ANNOTATION_TYPE
+ *
+ * Since: REPLACEME
+ */
+  HB_AAT_LAYOUT_SELECTOR_NO_ANNOTATION                   = 0,
+  HB_AAT_LAYOUT_SELECTOR_BOX_ANNOTATION                  = 1,
+  HB_AAT_LAYOUT_SELECTOR_ROUNDED_BOX_ANNOTATION          = 2,
+  HB_AAT_LAYOUT_SELECTOR_CIRCLE_ANNOTATION               = 3,
+  HB_AAT_LAYOUT_SELECTOR_INVERTED_CIRCLE_ANNOTATION      = 4,
+  HB_AAT_LAYOUT_SELECTOR_PARENTHESIS_ANNOTATION          = 5,
+  HB_AAT_LAYOUT_SELECTOR_PERIOD_ANNOTATION               = 6,
+  HB_AAT_LAYOUT_SELECTOR_ROMAN_NUMERAL_ANNOTATION        = 7,
+  HB_AAT_LAYOUT_SELECTOR_DIAMOND_ANNOTATION              = 8,
+  HB_AAT_LAYOUT_SELECTOR_INVERTED_BOX_ANNOTATION         = 9,
+  HB_AAT_LAYOUT_SELECTOR_INVERTED_ROUNDED_BOX_ANNOTATION = 10,
+/*
+ * Selectors for #HB_AAT_LAYOUT_FEATURE_TYPE_KANA_SPACING_TYPE
+ *
+ * Since: REPLACEME
+ */
+  HB_AAT_LAYOUT_SELECTOR_FULL_WIDTH_KANA   = 0,
+  HB_AAT_LAYOUT_SELECTOR_PROPORTIONAL_KANA = 1,
+
+/*
+ * Selectors for #HB_AAT_LAYOUT_FEATURE_TYPE_IDEOGRAPHIC_SPACING_TYPE
+ *
+ * Since: REPLACEME
+ */
+  HB_AAT_LAYOUT_SELECTOR_FULL_WIDTH_IDEOGRAPHS   = 0,
+  HB_AAT_LAYOUT_SELECTOR_PROPORTIONAL_IDEOGRAPHS = 1,
+  HB_AAT_LAYOUT_SELECTOR_HALF_WIDTH_IDEOGRAPHS   = 2,
+/*
+ * Selectors for #HB_AAT_LAYOUT_FEATURE_TYPE_UNICODE_DECOMPOSITION_TYPE
+ *
+ * Since: REPLACEME
+ */
```

Comment:
All the since tags should just go in the gtk-doc block for the enum.

Reply:
It was just a draft, now fixed.